.env.sample

```
@@ -1,6 +1,3 @@
-VITE_API_URL=http://localhost:1234
+// .env 파일 만들때 참고하세요
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The // .env 파일 만들때 참고하세요 key has incorrect delimiter

(IncorrectDelimiter)